<launch>
  <node pkg="jsk_rviz_plugins"
        type="overlay_sample.py"
        respawn="true"
        name="overlay_sample" />
  <node pkg="image_publisher"
        type="image_publisher"
        name="image_publisher"
        args="$(find rviz)/images/splash.png" />
  <node pkg="jsk_rviz_plugins"
        type="overlay_menu_sample.py"
        respawn="true"
        name="overlay_menu_sample" />
  <node pkg="jsk_rviz_plugins"
        type="rosconsole_overlay_text.py"
        respawn="true"
        name="rosconsole_overlay_text">
    <rosparam>
      reverse_lines: false
    </rosparam>
  </node>
  <node pkg="rostopic"
        type="rostopic"
        name="sample_string_publisher"
        args="pub -r 1 /sample_string std_msgs/String
              'This is a sample message of type std_msgs/String.'"/>
  <node pkg="rviz" type="rviz" name="rviz"
        output="screen"
        args="-d $(find jsk_rviz_plugins)/cfg/overlay_sample.rviz"/>
</launch>
